validate/setup-guides/do/sgDroplet.md

```diff
@@ -16,8 +16,8 @@ import DocsCards from '@components/global/DocsCards';
 
 ## Setup Digital Ocean Firewall Rules
 
-:::danger VERY IMPORTANT FOR MAINNET 2.0 LAUNCH
-These documents reference creating a single layer 0 or Metagraph.  For the launch of MainNet 2.0, you will installing a dual layer - `layer0` and `layer1` - validator Node.  You will need to add some extra firewall rules to accommodate this initial requirement.   Please refer to the end of this documentation for these additions.  
+:::danger VERY IMPORTANT FOR HYPERGRAPH OR METAGRAPH LAUNCH
+These documents reference creating a single layer 0 Hypergraph or Metagraph.  You will installing a dual layer - `layer0` and `layer1` - Hybrid Validator Node.  You will need to add some extra firewall rules to accommodate this initial requirement.   Please refer to the end of this documentation for these additions.  
```

Comment:
There are several of these but are these warnings still needed? Assuming anyone that reads these sections would have installed after MainNet 2 launch or have already upgraded (so not relevant). I would remove if these are otherwise covered properly in the documentation.